supertokens_python/recipe/thirdparty/providers/config_utils.py

```diff
@@ -143,15 +143,13 @@ def merge_config(
 
 
 def merge_providers_from_core_and_static(
-    tenant_id: Optional[str],
     provider_configs_from_core: List[ProviderConfig],
     provider_inputs_from_static: List[ProviderInput],
```

Comment:
the type of `ProviderConfig` and `ProviderConfigForClientType` still take a tenant_id in provider.py -> remove them.

Reply:
Please double check that the types are correct for these two classes as per the node SDK.

Reply:
Done